metrics_utility/automation_controller_billing/collectors.py

```diff
@@ -102,21 +103,21 @@ def get_install_type():
 
 @register('config', '1.0', description=_('General platform configuration.'), config=True)
 def config(since, **kwargs):
-    license_info = get_license()
+    license_info, settings_info = get_config_and_settings_from_db()
     return {
         'platform': {
             'system': platform.system(),
             'dist': distro.linux_distribution(),
             'release': platform.release(),
             'type': get_install_type(),
         },
-        'install_uuid': settings.INSTALL_UUID,
-        'instance_uuid': settings.SYSTEM_UUID,
-        'controller_url_base': settings.TOWER_URL_BASE,
-        'controller_version': get_awx_version(),
-        'license_type': license_info.get('license_type', 'UNLICENSED'),
+        'install_uuid': settings_info.get('install_uuid'),
+        'instance_uuid': settings_info.get('system_uuid', '00000000-0000-0000-0000-000000000000'),
+        'controller_url_base': settings_info.get('tower_url_base'),
+        'controller_version': get_controller_version_from_db(),
+        'license_type': license_info.get('license_type'),
```

Comment:
```suggestion
        'license_type': license_info.get('license_type', 'UNLICENSED'),
```

(missing the original default value)